### PR TITLE
chore: added liveness check

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,4 +10,4 @@ ENV PYTHONPATH=${PYTHONPATH}:'/app'
 RUN python3 /app/confd/generate-config.py
 RUN mkdir /app/src/outputs
 WORKDIR /app/src
-CMD ["python3", "app.py"]
+CMD ["python3", "main..py"]
```
Comment (cl-shaharshavit-rnd) [inline Dockerfile]: double dot

File: src/main.py [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
         self.__taskHandler = TaskHandler()
+        probe.readiness = True
 
     def _start_service(self):
         self.logger.info(f'Service is listening to broker: {self.__config["kafka"]["host_ip"]},'f' topic: {self.__config["kafka"]["topic"]}')
         try:
             self.__taskHandler.handle_tasks()
         except Exception as e:
             self.logger.error(f'Error occurred during running service: {e}')
+            probe.readiness = False
```
Comment (asafMasa) [inline src/main.py]: liveliness

Comment (asafMasa) [reply src/main.py]: add rediness

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply Dockerfile]: fixed in 5d9a038

File: src/main.py (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/main.py]: fixed in 5d9a038

**Review by asafMasa — APPROVED** [review]

**Pull request merged**